### ISSUE TITLE
Hostile mobs have wallhacks

### ISSUE THREAD

**Issue by WickedCybs** [Aurorastation/Aurora.3#15747]
Their sight isn't blocked by obstacles and they rapidly attack windows between them and a potential target even if they never sighted them previously. The attacks also happen at a far increased rate, at least two per second seems like.

This is especially getting bad with carp since that leads to a lot of broken windows.

**Comment by SleepyGemmy** [comment]
duplicate of #15621.
